packages/browser/src/posthog-surveys.ts

```diff
@@ -37,7 +37,7 @@ export class PostHogSurveys {
     onRemoteConfig(response: RemoteConfig) {
         // only load surveys if they are enabled and there are surveys to load
         const surveys = response['surveys']
-        if (isNullish(surveys)) {
+        if (isNullish(surveys) && !this._instance.config.disable_surveys) {
```

Comment:
not sure if this is needed since `onRemoteConfig` should not even be called if `disable_surveys=true` unless this is only for tests not respecting `disable_surveys` and calling it directly